### DISCUSSION TITLE
rate limit exceeded error

### DISCUSSION THREAD

**Discussion by sujeet036** [community/community#64310] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

![Git](https://github.com/community/community/assets/77746304/6f6edb4f-7981-4969-b5b9-37832877f978)

Hi,

When I tried to push the code, I got a rate limit exceeded error. Earlier, I was not getting that error.
Let me know how to avoid this. If I am uploading 35–40 files, I am getting this error.

**Comment by byrneh** [comment]
normally rate limiting is experienced with API usage.
If you are experiencing the problem with git push does your scenario include a number of LFS objects which are links, resulting in you hitting the limits?
[#rate-limiting](https://docs.github.com/en/rest/overview/resources-in-the-rest-api?apiVersion=2022-11-28#rate-limiting)
